src/main/java/nl/andrewl/emaildatasetbrowser/view/SwingUtils.java

```diff
@@ -27,22 +27,15 @@ public static void setAllButtonsEnabled(Container c, boolean enabled) {
 
 	@SuppressWarnings("unchecked")
 	public static <T extends Component> T findFirstInstance(Container c, Class<T> type) {
-		for (var component: c.getComponents()) {
+		for (var component : c.getComponents()) {
 			if (component.getClass().equals(type)) {
 				return (T) component;
 			} else if (component instanceof Container nested) {
 				T result = findFirstInstance(nested, type);
-				if (result != null) return result;
+				if (result != null)
+					return result;
```

Comment:
If you're going with the expanded syntax, I would prefer adding curly brackets as well.